### DISCUSSION TITLE
How to navigate between different panels

### DISCUSSION THREAD

**Discussion by AutobookNft** [filamentphp/filament#11957] [Help]
### Package

Panel builder

### Package Version

3.2.42

### How can we help you?

Hello everyone and thank you very much for any help you can give me,

I have three different panels, with three different configuration files, each has its own specific ->path() how can I navigate from one to the other?
```

class Dashboard extends \Filament\Pages\Dashboard
{
     protected static ?string $title = 'Home page';
     protected static string $routePath = 'app/overview';
     protected static ?int $navigationSort = -2;
}
```
this takes me to http://127.0.0.1:8000/utilities/app/overview
because in the configuration file I have ->path('utilities');
while I would like to be able to get to:
http://127.0.0.1:8000/app/overview

Basically I would like to find a way to navigate between the various Panels I have created

**Comment by VincvanAs** [comment]
Hello, maybe this plugin could assist you with this.

https://filamentphp.com/plugins/bezhansalleh-panel-switch

**Reply by AutobookNft** [reply]
wooow! 
Works great!
thank you with all my heart!
